platypus/parse.py

```diff
@@ -224,19 +226,24 @@ def process_results(percentage_ids, alignment_lengths, percentage_ids_other,
         List of dictionaries with the summarized results.
     """
     results = []
+    summary_fh = {}
 
     iter_a = product(percentage_ids, alignment_lengths)
     iter_b = product(percentage_ids_other, alignment_lengths_other)
+
     for (perc_id_a, aln_len_a), (perc_id_b, aln_len_b) in izip(iter_a, iter_b):
         filename = "p1_%d-a1_%d_p2_%d-a2_%d" % (perc_id_a, aln_len_a,
                                                 perc_id_b, aln_len_b)
+        summary_filename = join(output_dir, "summary_" + filename + ".txt")
+        summary_fh = open(summary_filename, 'w')
```

Comment:
I think this would be a good idea specially, if the number of files grows, we might run out of file handles :scream: